src/fielddef.ts

```diff
@@ -143,6 +143,7 @@ export interface FieldDefBase<F> {
 export interface FieldDef<F> extends FieldDefBase<F> {
   /**
    * The encoded field's type of measurement (`"quantitative"`, `"temporal"`, `"ordinal"`, or `"nominal"`).
+   * It can also be a geo type (`"latitude"`, `"longitude"`, and `"geojson"`) when applicable.
```

Comment:
Can you describe specifically what "when applicable" mean?  (So that users understand right away and do not have to try to find other docs to read.)

Reply:
I don't think this is a good idea. Adding anything more is just weirdly specific. Here's what I came up with, but it's honestly more confusing.
```
   * It can also be a geo type (`"latitude"`, `"longitude"`, and `"geojson"`) when projecting geographical coordinates using a `point` mark (on `x`, and `y` channels), a `rule` mark (on `x`, `x2`, `y`, and `y2` channels) or projecting a GeoJSON shapefile on a `geoshape` mark (on the `shape` channel).
```

Reply:
Cut this part -- it's too much

> using a `point` mark (on `x`, and `y` channels), a `rule` mark (on `x`, `x2`, `y`, and `y2` channels) or projecting a GeoJSON shapefile on a `geoshape` mark (on the `shape` channel)." 

and then paraphrase this part to be slightly human-friendly and link to related concept

> when projecting geographical coordinates

Reply:
I don't really understand what you're saying

Reply:
Sorry, I mean something like this:

> It can also be a geographic type (`"latitude"`, `"longitude"`, and `"geojson"`) for using with geographic projection

and link projection to projection docs 


("when applicable" is vague and doesn't convey much information :D)